tests/edgeless/text.spec.ts

```diff
@@ -237,8 +246,10 @@ test('auto wrap text by dragging left and right edge', async ({ page }) => {
   await page.mouse.up();
 
   // the text should be unwrapped, check the height of text element
-  await waitNextFrame(page);
-  await assertEdgelessSelectedRect(page, [96, 140, 94, 38]);
+  selectedRect = await getEdgelessSelectedRect(page);
+  expect(selectedRect.width).toBeGreaterThan(lastWidth);
+  expect(selectedRect.height).toBeLessThan(lastHeight);
+  // await assertEdgelessSelectedRect(page, [96, 140, 94, 38]);
```

Comment:
Maybe it's better to remove these commented lines, unless you can make it explicit what happens when you uncomment them.

Reply:
OK, I will remove them.

Reply:
I have remove some comments.